### PR TITLE
Embedded code generator violet

### PR DESCRIPTION
**Pull request by ChristophStanik**

### CODE REVIEW

**Comment by lazarDevelopment** [comment]
Review in Progress

File: client/assets/js/controller/MaterialDetailController.js [hunk 1]
```diff
@@ -23,12 +23,38 @@ MaterialDetailController.$inject = ['$scope', '$stateParams', '$state', '$contro
  * @param {stateParams} $stateParams
  * @param {state} $state
  */
-function MaterialDetailController($scope, $stateParams, $state, $controller, MaterialService, ContentService) {
+function MaterialDetailController($scope, $stateParams, $state, $location, $controller, MaterialService, ContentService) {
   'use strict';
-  angular.extend(this, $controller('DefaultController', {$scope: $scope, $stateParams: $stateParams, $state: $state}));
+  angular.extend(this, $controller('DefaultController', {$scope: $scope, $stateParams: $stateParams, $state: $state, $location: $location}));
```
Comment (lazarDevelopment) [inline client/assets/js/controller/MaterialDetailController.js]: Is "$location: $location" injection into the defaultcontroller necessary? Funktioniert auch ohne.

File: client/assets/js/controller/MaterialDetailController.js [hunk 2]
```diff
@@ -23,12 +23,38 @@ MaterialDetailController.$inject = ['$scope', '$stateParams', '$state', '$contro
...
   var contents = [];
   var contentId = 0;
+  var absUrl = $location.absUrl();
+  $scope.embedCode = '<iframe src="'+absUrl+'?embedded=true"  style="overflow:hidden;height:100vh;width:100%;"/>';
+  $scope.isEmbedded = $location.search().embedded;
+
+  function checkEmbedded() {
+    if($scope.isEmbedded === 'true') {
+      console.log("entered checkEmbedded with true");
+      angular.element(document.querySelector("#topBar")).addClass("hide");
```
Comment (lazarDevelopment) [inline client/assets/js/controller/MaterialDetailController.js]: please remove console.log

File: client/assets/js/controller/MaterialDetailController.js [hunk 3]
```diff
@@ -23,12 +23,38 @@ MaterialDetailController.$inject = ['$scope', '$stateParams', '$state', '$contro
...
+    }
+    else {
+      console.log("entered checkEmbedded with false");
```
Comment (lazarDevelopment) [inline client/assets/js/controller/MaterialDetailController.js]: please remove console.log

**Comment by ChristophStanik** [comment]
@lazarDevelopment I removed the console logs and '$location: $location'

**Comment by lazarDevelopment** [comment]
Review Completed: OK!

**Pull request merged**